LICENSES/README.md

```diff
@@ -0,0 +1,85 @@
+# systemd Project Licensing
+
+## Main License
+
+The systemd project uses a single-line reference to Unique License Identifiers
+in source files as defined by the Linux Foundation's SPDX project (https://spdx.org/).
```

Comment:
The lack of plural in "reference" somehow seems wrong. And I think it's important to say that the reference identifies the license for that particular file. (It seems obvious to us, but it seems better to say this clearly.)

Maybe "The systemd project uses single-line references to Unique Licese Identifiers as defined by the Linux Foundation's SPDX project (https://spdx.org/). The line in each individual source file identifies the license applicable to that file."